druid-shell/src/platform/windows/window.rs

```diff
@@ -1234,7 +1214,12 @@ impl WindowBuilder {
                 present_strategy: self.present_strategy,
             };
 
+            let position = match self.position {
+                Some(pos) => pos,
+                None => Point::new(CW_USEDEFAULT as f64, CW_USEDEFAULT as f64),
```

Comment:
I am slightly uncomfortable with CW_USEDEFAULT being converted into f64. I know it's going to work, but feels like it's not quite the right type for the job.

Would it work for `position` to be `(i32, i32)` here?

Reply:
Good catch, makes more sense to cast self.position to i32